mindjunkies/live_classes/tests/test_models.py

```
@@ -1,59 +1,61 @@
 import pytest
 from django.conf import settings
-from django.utils.timezone import now
+from django.utils.timezone import now, make_aware
 from freezegun import freeze_time
 from model_bakery import baker
+from datetime import datetime
 
 from mindjunkies.accounts.models import User
 from mindjunkies.courses.models import Course
 from mindjunkies.live_classes.models import LiveClass
+from model_bakery import baker
```

Ruff (0.8.2)

Redefinition of unused `baker` from line 5

Remove definition: `baker`

(F811)